shuffle/cli/src/shared.rs

```diff
@@ -50,6 +50,10 @@ pub fn get_home_path() -> PathBuf {
         .to_path_buf()
 }
 
+pub fn get_shuffle_networks_path() -> PathBuf {
+    get_shuffle_dir().join("networks")
```

Comment:
As mentioned above, let's swap this out with `home.get_networks_dir()` or `home.get_shuffle_dir().join(networks)`. In my forge tests, I have a different shuffle base than ~/.shuffle